dkpro-core-io-brat-asl/src/main/java/de/tudarmstadt/ukp/dkpro/core/io/brat/internal/model/BratTextAnnotation.java

```diff
@@ -44,11 +44,11 @@
     private final String text;
 
     public BratTextAnnotation(int aId, String aType, int aBegin, int aEnd, String aText)
-    {
-        this("T" + aId, aType, aBegin, aEnd, aText);
+    {    	
+        this("T" + String.format("%04d", aId), aType, aBegin, aEnd, aText);
```

Comment:
@joancf You mentioned in the previous PR this change is done to permit sorting in the JS (btw. the change to the sorting in the JS code is not included in the present PR anymore). But using the padded numbers is kind of brittle as one needs to pre-suppose the number of annotations. How about instead changing the sorting code so that it strips the annotation type letter and then compares the rest of the ID numerically?

Reply:
Also we could have an ID of the form  "layer-id"  for sorting, so it will sort by layer (the layer can be a two digits padded number, more than 100 layers would be crazy) 
for example 01-1 01-2 01-3 02-4 02-05 ....